### ISSUE TITLE
Add Service Worker doc/example for SignalR

### ISSUE THREAD

**Issue by bradygaster** [aspnet/AspNetCore.Docs#11921]
If we feel the browser service worker scenario warrants an entire doc article as requested via [this issue](https://github.com/aspnet/SignalR-samples/issues/18), use @davidfowl's code samples below to source an article on the topic. 

https://github.com/aspnet/AspNetCore/blob/9158039c9515e46d39c90f462ae91827769889c4/src/SignalR/samples/SignalRSamples/wwwroot/worker.html

https://github.com/aspnet/AspNetCore/blob/9158039c9515e46d39c90f462ae91827769889c4/src/SignalR/samples/SignalRSamples/wwwroot/worker.js

**Comment by davidfowl** [comment]
Corrected the issue (it's @halter73's code sample 😄 )

**Comment by anurse** [comment]
We should make sure to call out the limitations of using SignalR in a service worker. Because workers are spun up in response to an event and then shut down, you can’t leave a connection open throughout the life of the worker.

Most importantly, if you want the service worker to sit in the background waiting for the server to send it messages and then forward that data on to open Pages, it’s not trivial. This going to take some experimentation and exploration to get right :)

**Comment by davidfowl** [comment]
My bad I thought this was a web worker example not a service worker.

**Comment by anurse** [comment]
A web worker makes way more sense to me. Is that what was intended? Service workers aren't background services that can maintain a persistent connection.

**Comment by davidfowl** [comment]
Yes, this should be a web worker doc, not a service worker. I agree it doesn't make much sense talking about service workers.

**Comment by anurse** [comment]
Fixed :)
